lib/safira/accounts/accounts.ex

```diff
@@ -33,10 +33,27 @@ defmodule Safira.Accounts do
   end
 
   def create_user_uuid(attrs) do
+    Repo.transaction(fn ->
```

Comment:
DONT USER REPO TRANSACTION: 

To use Repo.transaction, do one of these:

    Either use bang functions everywhere in the transaction function (Repo.insert!, Repo.update! etc.), or…
    Use the with keyword and chain all the operations through non-bang functions and call Repo.rollback in the else clause – which would mean that the first failed operation will return {:error, reason} and the transaction will return that exact error so you can troubleshoot further afterwards. Or…
    Just use Ecto.Multi which will give you even more info if an operation fails. It’s really the best way of doing such composite operations ever since it was introduced. Just have one Ecto.Multi variable and append all your operations to it, then just execute it at once: Repo.transaction(your_multi).

Reply:
https://hexdocs.pm/ecto/Ecto.Multi.html#module-run: